Source/Scene/Model.js

```diff
@@ -2307,11 +2307,15 @@ define([
                     flipY : false
                 });
             }
-            // GLTF_SPEC: Support TEXTURE_CUBE_MAP.  https://github.com/KhronosGroup/glTF/issues/40
+        }
 
-            if (mipmap) {
-                tx.generateMipmap();
-            }
+        if (!defined(tx)) {
```

Comment:
Is this needed?  Could we just have

```
if (defined(tx) && mipmap) {
    tx.generateMipmap();
}
```

below?

Reply:
This is needed because we're using the `tx` value below.  So it seems like we're expecting it to be defined.
```
        model._rendererResources.textures[gltfTexture.id] = tx;
        model._texturesByteLength += tx.sizeInBytes;
```

Reply:
Maybe I am missing something - couldn't we just put a `defined(tx)` check below or even duplicate the code in both block above?  I am just trying to avoid the random `RuntimeError` to keep Coverity happy.

Reply:
@pjcozzi apparently I had it that way originally but @lilleyse said it should be a runtime error.  I changed it back to check defined